examples/amp-story/quiz.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
```

Comment:
Can you please make sure that this story is valid AMP?
http://validator.ampproject.org/